test/e2e/app/main.go

```diff
@@ -235,6 +250,7 @@ func startMaverick(cfg *Config) error {
 		misbehaviors[height] = mcs.MisbehaviorList[misbehaviorString]
 	}
 
+	// TODO: What is a maverick node?
 	n, err := maverick.NewNode(tmcfg,
```

Comment:
There was some outdated code in the maverick node too. Actually, what is a maverick node and why does it seem to be broken?

Reply:
a maverick node is used to test for byzantine actions. It's broken because the way the node is written is that it uses copied code from the main node. Since the main node has changed a lot I guess the maverick node was not updated properly with all the changes.